src/components/search/Filter/Filter.tsx

```diff
@@ -65,7 +65,12 @@ function Filter({
 
   let onDismissTransition: () => unknown
   const [activeFacets, setActiveFacets] = useState<ActiveFacets[]>([])
-  const filteredFacets = facets.filter((facet) => facet.type === 'BOOLEAN')
+  const filteredFacets = facets.filter((facet) => {
+    const hasOthersThanSlug = facet.values.some(({ value }) => value !== slug)
+    const isBoolean = facet.type === 'BOOLEAN'
+
+    return isBoolean && hasOthersThanSlug
+  })
```

Comment:
If you want the performance, you can leverage your code from the lazy evaluation behavior.

```tsx
// outside the component body
const checkFacetIsValid = <T>(facets: T[], verificationCallback: (T) => boolean): boolean => facets.some(verificationCallback)

// inside the component
return isBoolean && checkIfFacetsIsValid(facets.value, ({value}) => value !== slug))
```

Also, you can go further, assuming that this algorithm is complex (m*n), you can put it into the useMemo